### PR TITLE
Add AVS2 support

### PR DESCRIPTION
**Pull request by Dima73**
thanks @Captain

### CODE REVIEW

File: lib/python/Components/Converter/PliExtraInfo.py [hunk 1]
```diff
@@ -49,7 +49,7 @@
 	14: "DIVX 4",
 	15: "DIVX 5",
 	16: "AVS",
-	17: "N/A 17",
+	17: "AVC2",
```
Comment (persianpros) [inline lib/python/Components/Converter/PliExtraInfo.py]: Typo?

Comment (Dima73) [reply lib/python/Components/Converter/PliExtraInfo.py]: > 
> 
> Typo?

yes

**Comment by persianpros** [comment]
https://github.com/openatv/enigma2/commit/eff108a3bb2c7f9c114c68d503618e41d34d635f

**Comment by Dima73** [comment]
> 
> 
> [openatv/enigma2@eff108a](https://github.com/openatv/enigma2/commit/eff108a3bb2c7f9c114c68d503618e41d34d635f)
Why?

**Comment by Dima73** [comment]
Let's leave that for later.

**Pull request closed without merging**